appimage/server_config.json.standard

```diff
@@ -4,5 +4,8 @@
   "environment": {
     "server_config": "password",
     "deployment": "standard"
+  },
+  "mongodb": {
+        "start_mongodb": true
```

Comment:
Not exactly relevant to this PR, but I created tests and other configs with this mongodb param. Because it doesn't break anything and contributes to mongodb PR